src/Paket.Core/LockFile.fs

```diff
@@ -179,6 +179,10 @@ module LockFileSerializer =
                     | None -> ()
                     | Some command -> yield "      build: " + command
 
+                    match file.PackagePath with
+                    | None -> ()
+                    | Some path -> yield "      path: " + path
+
```

Comment:
ah sorry didn't understand the question. Yes It's parsed [here](https://github.com/fsprojects/Paket/blob/v3/src/Paket.Core/LockFile.fs#L336-L341) so that it can be later read [here](https://github.com/fsprojects/Paket/blob/v3/src/Paket.Core/RemoteDownload.fs#L291)